action.yml

```diff
@@ -117,6 +117,7 @@ runs:
         CREATE_PR: ${{ inputs.create-pr }}
       shell: bash
       run: |
```

Comment:
can you move to here
```
working-directory: ${{ inputs.working-dir }}
```

instead of using the cd?

thanks!

Reply:
and add a test case that run only in the `https://github.com/chainguard-dev/digestabot/tree/main/testfiles/test-dir` to validate